ops/run.sh

```diff
@@ -6,6 +6,13 @@ export TMHOME=/srv/near
 tendermint init
 cp /near/config.toml ${TMHOME}/config/config.toml
 
+if [[ -z ${PRIVATE_NETWORK} ]]
+then
+echo "Not running on a private network"
+else
+sed -i 's/addr_book_strict\ =\ true/addr_book_strict\ =\ false/g' ${TMHOME}/config/config.toml
```

Comment:
is there no way to override tendermint config by env vars or arguments? this is pretty gnarly

Reply:
I wasn't able to find the other way to do it. Only a small subset of the config can be overridden through flags.